pandas/tests/frame/test_sorting.py

```diff
@@ -227,6 +228,30 @@ def test_stable_descending_multicolumn_sort(self):
                                    kind='mergesort')
         assert_frame_equal(sorted_df, expected)
 
+    def test_sort_multi_index(self):
+        # GH 25775
+        s = ',a,b,c,d,e' + \
+            '0,0.548,7,0.602,0.544,0.423' + \
+            '1,0.645,4,0.891,0.963,0.383' + \
+            '2,0.791,5,0.568,0.925,0.071' + \
+            '3,0.087,0,0.832,0.778,0.870' + \
+            '4,0.978,7,0.461,0.780,0.118' + \
+            '5,0.639,1,0.944,0.521,0.414' + \
+            '6,0.264,7,0.456,0.568,0.018' + \
+            '7,0.617,6,0.616,0.943,0.681' + \
+            '8,0.359,4,0.697,0.060,0.666' + \
+            '9,0.670,2,0.128,0.315,0.363'
+        data = StringIO(s)
+        df1 = (pd.read_csv(data)
+               .assign(b=lambda df: (df.b * 10).astype(int)))
```

Comment:
these df1, df2 are not clear at all can you rename to something more descriptive.

Reply:
updated to the simple example by the other reviewer